app/src/main/java/com/example/pygmyhippo/user/MyEventsFragment.java

```diff
@@ -9,7 +9,7 @@
 import androidx.annotation.NonNull;
 import androidx.fragment.app.Fragment;
 
-import com.example.pygmyhippo.databinding.FragmentMyeventsBinding;
+import com.example.pygmyhippo.databinding.UserFragmentMyeventsBinding;
```

Comment:
NIT: naming should be UserFragmentMyEventsBinding?

Reply:
yes because there are two fragments my event binding, one for organiser and one for user

Reply:
oops should specify, did you mean for events to be capitalized or no

Reply:
this is auto generated from the xml file user_fragment_myevents.xml in the layout folder. are you suggesting i change the name of that xml file?